From 7f1083c838152bdb78329a3c5307a133a94f1849 Mon Sep 17 00:00:00 2001
From: Tobias Tefke <tobias.tefke@gmail.com>
Date: Wed, 28 Mar 2018 11:50:35 +0200
Subject: [PATCH] Export MTK paths

---
 rootdir/init.environ.rc.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rootdir/init.environ.rc.in b/rootdir/init.environ.rc.in
index 032e6987e..211dae5d6 100644
--- a/rootdir/init.environ.rc.in
+++ b/rootdir/init.environ.rc.in
@@ -11,3 +11,5 @@ on init
     export SYSTEMSERVERCLASSPATH %SYSTEMSERVERCLASSPATH%
     export LD_PRELOAD libsigchain.so%TARGET_LDPRELOAD%
     %EXPORT_GLOBAL_ASAN_OPTIONS%
+    export BOOTCLASSPATH $BOOTCLASSPATH:/system/framework/core-oj.jar:/system/framework/core-libart.jar:/system/framework/conscrypt.jar:/system/framework/okhttp.jar:/system/framework/core-junit.jar:/system/framework/bouncycastle.jar:/system/framework/ext.jar:/system/framework/framework.jar:/system/framework/telephony-common.jar:/system/framework/voip-common.jar:/system/framework/ims-common.jar:/system/framework/apache-xml.jar:/system/framework/org.apache.http.legacy.boot.jar:/system/framework/mediatek-common.jar:/system/framework/mediatek-framework.jar:/system/framework/mediatek-telephony-common.jar
+    export SYSTEMSERVERCLASSPATH $SYSTEMSERVERCLASSPATH:/system/framework/services.jar:/system/framework/ethernet-service.jar:/system/framework/wifi-service.jar
-- 
2.16.2

