From 3523542bf68b7b369e1febf369850177e3126ef0 Mon Sep 17 00:00:00 2001
From: Tobias Tefke <tobias.tefke@gmail.com>
Date: Sun, 8 Apr 2018 18:32:27 +0200
Subject: [PATCH 3/3] surfaceflinger: update display types for MTK

---
 services/surfaceflinger/Android.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/services/surfaceflinger/Android.mk b/services/surfaceflinger/Android.mk
index 18cc946be..0c67eac48 100644
--- a/services/surfaceflinger/Android.mk
+++ b/services/surfaceflinger/Android.mk
@@ -138,6 +138,10 @@ ifeq ($(BOARD_USE_BGRA_8888),true)
     LOCAL_CFLAGS += -DUSE_BGRA_8888
 endif
 
+ifeq ($(MTK_HWC_SUPPORT),true)
+    LOCAL_CFLAGS += MTK_HWC
+endif
+
 LOCAL_CFLAGS += -fvisibility=hidden -Werror=format
 LOCAL_CFLAGS += -std=c++14
 
-- 
2.16.3

